### PR TITLE
va-breadcrumb removed react breadcrumb component

### PR DESCRIPTION
**Pull request by harshil1793**
## Chromatic

https://1632-remove-breadcrumb-react-component--60f9b557105290003b387cd5.chromatic.com

---

## Description
This PR removes the `Breadcrumbs` react component.
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1632

## Testing done
Storybook

## Screenshots
N/A

## Acceptance criteria
- [ ] Deprecated component has been removed entirely.

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by jamigibbs — COMMENTED** [review]
I just want to confirm a couple of things before this is merged:

1. Have all instances of `Breacrumbs` been removed from vets-website?
2. I think we should create a new release after this is merged. If that's going to be done then the `core` version should updated and a release coordinated with the release manager.

**Comment by jamigibbs** [comment]
A couple other things we need to check before the ticket is closed: 

 - [ ] Remove any ESLint rules from the [eslint-plugin](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/master/packages/eslint-plugin).
- [ ] Remove the related web component(s) from the [web component migrations list](https://design.va.gov/about/developers/using-web-components#web-component-migrations)

**Comment by harshil1793** [comment]
@jamigibbs 

1. Yes all the instances have been removed.
2. I don't mind bumping core version and plan a release after merging.

The Rest of the tasks are ready and will be raising a PR soon.

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**
